ptero_lsf/implementation/backend.py

```diff
@@ -1,5 +1,6 @@
 from . import celery_tasks  # noqa
 from . import models
+from flask import request
```

Comment:
If we import flask here, anything that uses the backend.py will need to have flask installed in its environment.